fem/libceed/ceed.cpp

```diff
@@ -157,15 +577,7 @@ static void InitCeedNonTensorRestriction(const FiniteElementSpace &fes,
 {
```

Comment:
Looks like the integration rule is not needed here — can it be deleted from the argument list?

Reply:
Woops, that was my mistake. I think that should be fine.

Reply:
done.